flow/visualize/i210_replay.py

```diff
@@ -55,8 +58,9 @@ def replay(args, flow_params, output_dir=None, transfer_test=None, rllib_config=
     """
     assert bool(args.controller) ^ bool(rllib_config), \
         "Need to specify either controller or rllib_config, but not both"
-
-    if args.run_transfer:
+    if transfer_test is not None:
+        if type(transfer_test) == bytes:
+            transfer_test = ray.cloudpickle.loads(transfer_test)
```

Comment:
another stupid bug that took me a long time. ray can't pickle / unpickle the transfer test class for some reason. I had to do it on my own